src/main/java/com/to/ordermanagementservice/dto/OrderDetails.java

```diff
@@ -1,10 +1,21 @@
 package com.to.ordermanagementservice.dto;
 
-public class OrderDetails {
+import com.to.ordermanagementservice.entity.OrderItem;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+public class OrderDetails {//todo reterne all list of order and user ()
 
     private Integer orderId;
 
-    private Integer customerId;
+    private Integer userId;// todo put castomer id not user
```

Comment:
According to the https://github.com/Andrii77711/order-management-service/issues/1 it should be called `customerId`